pkg/mount/block/blockdev.go

```diff
@@ -165,6 +170,42 @@ func (b *BlockDev) ReadPartitionTable() error {
 	return unix.IoctlSetInt(int(f.Fd()), unix.BLKRRPART, 0)
 }
 
+// FindPCIInfo searches sysfs for the PCI vendor and device id.
+// We fill in the PCI struct with just those two elements.
+func (b *BlockDev) FindPCIInfo() (*pci.PCI, error) {
+	dPath := filepath.Join("/sys/class/block", b.Name, "device")
+	// Loop through devices until we find the actual backing pci device.
+	d, err := os.Stat(dPath)
+	for err == nil && d.IsDir() {
```

Comment:
why does this tell you what the actual backing pci device is? (comment in code?)

Reply:
Done